website/payments/templates/payments/payment_list.html

```diff
@@ -92,6 +92,9 @@ <h5 class="mb-4">
                 {% trans "You have not made any payments during this period." as no_payments %}
                 {% alert 'info' no_payments %}
             {% endif %}
+            <span class="mt-3 font-italic">
+                {% trans "Thalia only stores your payments for 7 years." %}
```

Comment:
Maybe add that this is a legal requirement?

Reply:
Meh, they can read it in the privacy policy. The reason I added this here was just to explain why you don't see everything.